# requested command
CartesianPositionCommandMsg command
---
# current command
CartesianPositionCommandMsg command